src/python/WMCore/WMBS/MySQL/Create.py

```diff
@@ -65,6 +65,26 @@ def __init__(self, logger = None, dbi = None, params = None):
           """CREATE UNIQUE INDEX uniq_wmbs_file_run_lumi on
              wmbs_file_runlumi_map (fileid, run, lumi)"""
 
+        #
+        # Insert LogCollect site to account for LogCollect output
+        #
+        self.inserts["99_location_logcollect_1"] = \
```

Comment:
It adds the output file to the FJR, so the JobAccountant puts it into WMBS. That's one of the things I want to talk to @ticoann about, whether we actually use that for anything (only thing I can think off is that WMArchive wants to have it in the FJR). We certainly don't need it for any further processing, as far as WMBS is concerned this is dead weight.